src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/client/transformation/TransformationClient.java

```diff
@@ -64,13 +64,18 @@ public SuccessfulTransformationResponse transformExceptionRecord(
             );
 
             if (ex.getStatusCode().equals(UNPROCESSABLE_ENTITY) || ex.getStatusCode().equals(BAD_REQUEST)) {
```

Comment:
This was supposed to be changed, but the behaviour is still incorrect - unprocessable entity should result in a 200 response with errors and warnings, but a bad request should result with a 500 response from us.